cmd/crowdsec/crowdsec.go

```diff
@@ -40,46 +41,75 @@ func initCrowdsec() (*parser.Parsers, error) {
 	return csParsers, nil
 }
 
-func runCrowdsec(parsers *parser.Parsers) error {
+func runCrowdsec(cConfig *csconfig.GlobalConfig, parsers *parser.Parsers) error {
 	inputLineChan := make(chan types.Event)
 	inputEventChan := make(chan types.Event)
 
 	//start go-routines for parsing, buckets pour and ouputs.
-	for i := 0; i < cConfig.Crowdsec.ParserRoutinesCount; i++ {
-		parsersTomb.Go(func() error {
-			defer types.CatchPanic("crowdsec/runParse")
-			err := runParse(inputLineChan, inputEventChan, *parsers.Ctx, parsers.Nodes)
-			if err != nil {
-				log.Fatalf("starting parse error : %s", err)
-				return err
+	parserWg := &sync.WaitGroup{}
```

Comment:
comment the reason of the waitgroup as it might not be obvious for future readers